src/test/groovy/pl/allegro/tech/build/axion/release/domain/VersionResolverTest.groovy

```diff
@@ -4,7 +4,7 @@ import pl.allegro.tech.build.axion.release.RepositoryBasedTest
 import pl.allegro.tech.build.axion.release.domain.properties.NextVersionProperties
 import pl.allegro.tech.build.axion.release.domain.properties.TagProperties
 import pl.allegro.tech.build.axion.release.domain.properties.VersionProperties
-import spock.lang.Ignore
+import spock.lang.Shared
```

Comment:
nitpick - this is now unused :) other than that +1

Reply:
Sorry, fixed!